include/core/surface_tension_model.h

```diff
@@ -0,0 +1,112 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#ifndef lethe_surface_tension_model_h
+#define lethe_surface_tension_model_h
+
+#include <core/physical_property_model.h>
+
+/**
+ * @brief SurfaceTensionModel. Abstract class that allows to calculate the
+ * surface tension.
+ */
+class SurfaceTensionModel : public PhysicalPropertyModel
+{
+public:
+  /**
+   * @brief Instantiates and returns a pointer to a SurfaceTensionModel object by casting it to
+   * the proper child class
+   *
+   * @param material_properties Parameters for a single fluid
```

Comment:
I think this line needs to be updated? It should be `surface_tension_parameters`, right?

Reply:
You're right ahahah a classic copy-paste and forget to edit. Thanks!